client/context/context.go

```diff
@@ -276,17 +278,17 @@ func (ctx CLIContext) PrintOutput(toPrint interface{}) error {
 }
 
 // GetFromFields returns a from account address and Keybase name given either
-// an address or key name. If genOnly is true, only a valid Bech32 cosmos
+// an address or key name. If offline is true, only a valid Bech32 cosmos
 // address is returned.
-func GetFromFields(input io.Reader, from string, genOnly bool) (sdk.AccAddress, string, error) {
+func GetFromFields(input io.Reader, from string, offline bool) (sdk.AccAddress, string, error) {
 	if from == "" {
 		return nil, "", nil
 	}
 
-	if genOnly {
+	if offline {
```

Comment:
Mmmm, no. This should actually check on a `genOnly` boolean

Reply:
@jgimeno the thing, is `generate-only` is true one you could even avoid calling `GetFromFields` altogether